src/controllers/StyleController.ts

```diff
@@ -1,230 +1,18 @@
-import type {
-    Display,
-    FlexColor,
-    FlexNumber,
-    GradeToOne,
-    Overflow,
-    Position,
-    Styles,
-    VerticalAlign,
-} from '../utils/types';
-import type {
-    TextStyleAlign,
-    TextStyleFontStyle,
-    TextStyleFontVariant,
-    TextStyleFontWeight,
-    TextStyleLineJoin,
-    TextStyleTextBaseline,
-    TextStyleWhiteSpace,
-} from '@pixi/text';
-import { TEXT_GRADIENT, TextStyle } from '@pixi/text';
-import { getColor } from '../utils/helpers';
-import { OVERFLOW, ALIGN, VERTICAL_ALIGN } from '../utils/constants';
+import type { GradeToOne, Styles } from '../utils/types';
```

Comment:
This controller is refactored a lot. Same logic but with aiming to more easy control over the properties. 

All the control now happens through the `set` method.